### PR TITLE
Fix bug 278 minor spelling issues

### PR DESCRIPTION
**Pull request by thomasjsweet**
Changes to spelling in two files
versioning/readme.md
CSharp.md

### CODE REVIEW

File: source-control/versioning/readme.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ Updating the major / minor version indicates changes of the API / interfaces / c
 
 ## Semantic Versioning
 
-Semantic Versioning is a concept of calculating the version number automatically based on a certain source code respository.
+Semantic Versioning is a concept of calculating the version number automatically based on a certain source code rspository.
```
Comment (debugerr) [inline source-control/versioning/readme.md:33]: repository :)

Comment (thomasjsweet) [reply source-control/versioning/readme.md]: How embarrassing for me.  Working on fixing - thank you

**Comment by thomasjsweet** [comment]
All spelling errors are fixed now.

**Comment by thomasjsweet** [comment]
Is there anything more for me to to do on this pull request?

**Comment by fnocera** [comment]
@thomasjsweet We usually leave it up to the person who opened the PR to complete in case they are still addressing comments but in this case I will complete. Just FYI for next time feel free to complete the PR yourself.

**Comment by thomasjsweet** [comment]
Thank you. It said I could not merge, as I did not have write permissions, which is fine btw.

**Pull request merged**